neuro_san/api/grpc/agent.proto

```diff
@@ -74,10 +74,17 @@ message Function {
     // Outward-facing description of what the agent does.
     string description = 1 [json_name="description"];
 
-    // Optional map of parameters that the agent needs in order to work.
+    // Optional map of parameters passed in via the natural-language chat text channel
+    // that the agent needs in order to work.
     // This is really a pydantic/OpenAI function description, which is
     // a bit too complex to specify directly in protobuf.
     google.protobuf.Struct parameters = 2 [json_name="parameters"];
+
+    // Optional map of parameters passed in via the sly_data dictionary private data channel
+    // that the agent needs in order to work.
+    // This is really a pydantic/OpenAI function description, which is
+    // a bit too complex to specify directly in protobuf.
+    google.protobuf.Struct sly_data = 3 [json_name="sly_data"];
```

Comment:
Add sly_data schema to the Function spec returned.

Reply:
Good point - `sly_data` is confusing. `sly_data_schema` or `expected_sly_data_schema` would be better

Reply:
Done.